ts/smelter-browser-render/src/wasm.ts

```diff
@@ -4,8 +4,15 @@ import init, * as wasm from './generated/smelter/compositor_web';
  * Loads and initializes wasm module required for the smelter to work.
  * @param wasmModuleUrl {string} - An URL for `smelter.wasm` file. The file is located in `dist` folder.
  */
-export async function loadWasmModule(wasmModuleUrl: string) {
-  await init({ module_or_path: wasmModuleUrl });
-}
+export const loadWasmModule = (() => {
```

Comment:
when this happens? wasm is loaded in worker code, so there should never be a case when this happens unless someone is using browser-render directlly?

Reply:
Correct, it only happens in the Counter example which uses the renderer directly